src/frontend/src/pages/CreateWorkPackagePage/CreateWorkPackageFormView.tsx

```diff
@@ -0,0 +1,237 @@
+  const query = useQuery();
+
+  const {
+    handleSubmit,
+    control,
+    formState: { errors },
+    register
+  } = useForm({
+    resolver: yupResolver(schema),
+    defaultValues: {
+      name: '',
+      wbsNum: query.get('wbs') || '',
+      crId: Number(query.get('crId')),
+      startDate: new Date(),
+      duration: null,
+      dependencies: [] as { wbsNum: string }[],
+      expectedActivities: [] as { bulletId: number; detail: string }[],
+      deliverables: [] as { bulletId: number; detail: string }[]
+    }
+  });
+
+  const {
+    fields: expectedActivities,
+    append: appendExpectedActivity,
+    remove: removeExpectedActivity
+  } = useFieldArray({ control, name: 'expectedActivities' });
+  const {
+    fields: deliverables,
+    append: appendDeliverable,
+    remove: removeDeliverable
+  } = useFieldArray({ control, name: 'deliverables' });
+  const {
+    fields: dependencies,
+    append: appendDependency,
+    remove: removeDependency
+  } = useFieldArray({ control, name: 'dependencies' });
+
+  return (
+    <form
+      id={'create-work-package-form'}
+      onSubmit={(e) => {
+        e.preventDefault();
+        e.stopPropagation();
+        handleSubmit(onSubmit)(e);
+      }}
+      onKeyPress={(e) => {
+        e.key === 'Enter' && e.preventDefault();
+      }}
+    >
+      <PageTitle title={'New Work Package'} previousPages={[{ name: 'Work Packages', route: routes.PROJECTS }]} />
+      <PageBlock title={''}>
+        <Grid container spacing={2}>
+          <Grid item xs={12} md={6}>
+            <FormControl sx={{ width: '100%' }}>
+              <FormLabel>Work Package Name</FormLabel>
+              <ReactHookTextField
+                name="name"
+                control={control}
+                placeholder="Enter work package name..."
+                errorMessage={errors.name}
+              />
+            </FormControl>
+          </Grid>
+          <Grid item xs={12} md={6}>
+            <FormControl>
+              <FormLabel>Change Request ID</FormLabel>
+              <ReactHookTextField
+                name="crId"
+                control={control}
+                placeholder="Enter change request ID..."
+                errorMessage={errors.crId}
+                type="number"
+              />
+            </FormControl>
+          </Grid>
+          <Grid item xs={12} md={3}>
+            <FormControl>
+              <FormLabel>Project WBS Number</FormLabel>
+              <ReactHookTextField
+                name="wbsNum"
+                control={control}
+                placeholder="Enter project WBS number..."
+                errorMessage={errors.wbsNum}
+              />
+            </FormControl>
+          </Grid>
+          <Grid item xs={12} md={3}>
+            <Controller
+              name="startDate"
+              control={control}
+              rules={{ required: true }}
+              render={({ field: { onChange, value } }) => (
+                <FormControl>
```

Comment:
The FormControl/ Label goes outside the controller

Reply:
also the date picker looks a bit weird in I think the medium view:
![image](https://user-images.githubusercontent.com/29521172/211241922-df2e76f0-688e-4cdb-8626-66a71ef95e6e.png)